vllm/model_executor/layers/mamba/ops/ssd_chunk_scan.py

```diff
@@ -442,7 +442,7 @@ def _chunk_scan_fwd_kernel(
              (offs_out_n[None, :] < hdim))
 
 
-def _seq_idx_to_chunk_indices_offsets(seq_idx, chunk_size: int):
+def seq_idx_to_chunk_indices_offsets(seq_idx, chunk_size: int):
```

Comment:
Since we need to call this in the model definition, it might be nice to wrap the output in a dataclass (something like `Mamba2Metadata`) and create a helper function to construct thi sin `mamba_mixer2.py`. Just to have the code self-document this low-level detail

Reply:
moved to mamba2_metadata.py